recipes/r-allelicseries/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '0.0.3.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-allelicseries
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/AllelicSeries_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/AllelicSeries/AllelicSeries_{{ version }}.tar.gz
+  sha256: ff0a07553d400d1560576bf41ae98f8270c4831c84f2e36e6cf4d863f0b409f2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/r.dll'        # [win]
+    - '*/Rlapack.dll'  # [win]
+    - '*/Rblas.dll'    # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - libblas
+    - liblapack
+    - _openmp_mutex                # [linux]
```

I believe canonical requirement is `libgomp` for this:

```suggestion
    - libgomp                      # [linux]
```